VaticanBAV/et/BAVet143.xml

```diff
@@ -0,0 +1,388 @@
+                                    </support>
+                                    
+                                    <extent>
+                                        <measure unit="leaf" n="1">1</measure>
+                                        <locus target="#188"/>
+                                    </extent>
+                                    <condition>The leaf was removed from another codex and inserted into this manuscript as a guardleaf.</condition>
+                                </supportDesc>
+                            </objectDesc>
+                            
+                            <handDesc>
+                                <handNote script="Ethiopic" xml:id="h2">
+                                    <date notAfter="1928">The cataloguers have provided no information on this specific hand.</date>                                   
+                                </handNote>
+                            </handDesc>
+                            
+                            <additions>
+                                <list>
+                                    <item xml:id="a1">
+                                        <locus target="#188r"/>
+                                        <desc type="Supplication">
+                                            Short invocation written by multiple hands in the upper margin of the folio. Part of the text is illegible or lost. 
+                                            The names of <persName ref="PRS15039SayfaAm">Sayfa ʾAmlāk</persName> and 
+                                            <persName ref="PRS15040BetaGi">Beta Giyorgis Sirāk</persName> are mentioned as <foreign xml:lang="gez">ገብርከ፡</foreign>, ‘your servant’. 
+                                        </desc>
+                                        <q xml:lang="gez">
+                                            <gap reason="lost"/> ሰውሮ፡ እም<gap reason="lost"/> ውግዓት፡ አድኅኖ፡ ለገብርከ፡ ሰይፈ፡ አም<lb/>ላክ፡ 
+                                            <gap reason="lost"/> ዕጣን፡ ሀብተ፡ ማርያም<supplied reason="omitted">፡</supplied> 
+                                            <lb/> <gap reason="lost"/> ለገብርከ፡ ቤተ፡ ጊዮርጊስ፡ ሲራክ፡
+                                        </q>
+                                    </item>
+                                </list>
+                            </additions>
+                        </physDesc>                  
+                    </msPart>
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <particDesc>
+                <listPerson>
+                    <person>
+                        <persName ref="PRS15037Falaski" role="owner"/>
+                    </person>
+                    <person>
+                        <persName ref="PRS15038WaldaMa" role="owner"/>
+                    </person>
+                    <person role="scribe">
```

Comment:
it is better to avoid empty elements, they may cause issues in the app which usually expects some content